math/prime_test.py

```diff
@@ -15,4 +15,22 @@ def prime_test(n):
         if n%(j)==0 or n%(j+2)==0:
             return False
         j += 6
-    return True
\ No newline at end of file
+    return True
+
+
+def prime_test(n):
+    # prime numbers are greater than 1
+    if num > 1:
+    # check for factors
+        for i in range(2,num):
```

Comment:
The range can be (2, sqrt(num)+1) for more efficiency.
```python
for i in range(2, int(num ** 0.5) + 1):
```

Reply:
Yeah, that is correct. Why don't you approve it if It is correct?